loader/extension_manual.c

```diff
@@ -193,7 +194,7 @@ VKAPI_ATTR VkResult VKAPI_CALL terminator_AcquireXlibDisplayEXT(VkPhysicalDevice
         return VK_ERROR_INITIALIZATION_FAILED;
     }
 }
-
+#endif // VULKANSC
```

Comment:
The #if is define above the PLATFORM_XLIB_XRANDR but this is defined inside of it.

Reply:
fixed.